src/main/java/kitchenpos/domain/Product.java

```diff
@@ -7,6 +7,14 @@ public class Product {
     private String name;
     private BigDecimal price;
 
+    public Product() {
```

Comment:
```suggestion
    public Product() {}
```

이 부분은 중요한 것은 아니지만 [컨벤션](https://naver.github.io/hackday-conventions-java/#permit-concise-empty-block)에 따르면 이렇게 작성할 수도 있을 것 같습니다..!